### ISSUE TITLE
Add doctype to the index.html we provide

### ISSUE THREAD

**Issue by agcolom** [jquery/themeroller.jquerymobile.com#226]
I think we should do this. Would this go here? 
https://github.com/jquery/themeroller.jquerymobile.com/blob/master/inc/index.html.inc

We have inconsistencies also between <!doctype html> and <!DOCTYPE html> in places... which one should we go for? (I have a preference for <!doctype html>)
cc: @scottgonzalez @arschmitz @jaspermdegroot @cgack @gabrielschulhof

**Comment by scottgonzalez** [comment]
All lowercase: http://contribute.jquery.org/style-guide/html/#doctype

**Comment by jaspermdegroot** [comment]
Yeah, we should add the doctype back. It got lost with [this commit](https://github.com/jquery/themeroller.jquerymobile.com/commit/14e354350d1b799d3d73f308075f681754a0d042).
